sklearn/kernel_ridge/krr.py

```diff
@@ -0,0 +1,104 @@
+# Authors: Mathieu Blondel <mathieu@mblondel.org>
+#          Jan Hendrik Metzen <jhm@informatik.uni-bremen.de>
+# License: BSD 3 clause
+
+from ..base import RegressorMixin
+from .kernel_ridge import KernelRidgeBase
+
+
+class KernelRidgeRegression(RegressorMixin, KernelRidgeBase):
```

Comment:
This is currently called KernelRidge and should stay that way

Reply:
Done